vsphere/datadog_checks/vsphere/vsphere.py

```diff
@@ -920,7 +681,7 @@ def collect_metrics(self, instance):
             if mor['mor_type'] == 'vm':
                 vm_count += 1
             if 'metrics' not in mor or not mor['metrics']:
-                # self.log.debug("Skipping entity %s collection because we didn't cache its metrics yet" % mor['hostname'])
+                # self.log.debug("Skipping entity %s collection b/c we didn't cache its metrics yet" % mor['hostname'])
```

Comment:
Why not just kill this line?

Reply:
lol, blindly shortened the line to make flake8 happy, too long didn't read.
Wiping it...